overcast_to_sqlite/datastore.py

```
@@ -447,3 +457,22 @@
             }
             for result in results
         ]
+
+    def cleanup_old_episodes(self) -> None:
+        """Delete episodes older than OVERCAST_LIMIT_DAYS if more than 100 episodes exist."""
```

Ruff (0.11.9)

Line too long (93 > 88)

(E501)